compose/cli/main.py

```diff
@@ -233,7 +233,7 @@ class TopLevelCommand(object):
       top                Display the running processes
       unpause            Unpause services
       up                 Create and start containers
-      version            Show the Docker-Compose version information
+      version            Show version of Compose and its dependencies
```

Comment:
```suggestion
      version            Show version of Docker-Compose and its dependencies
```

Reply:
In my day "Docker Compose" was the software, "Compose" was a short hand, and `docker-compose` was only ever used in documentation to refer to the command you were calling. Maybe that's changed? Either way, surely it shouldn't be "Docker-Compose" (capitalized with a dash)? :)

Reply:
Maybe to side step this problem it could just be "Show version of this command and its dependencies", haha.

Or, just "Show version".

Reply:
Actually, [the documentation says this is still the case](https://docs.docker.com/compose/), unless the documentation is wrong:

> Compose is a tool for defining and running multi-container Docker applications.

A tool, not a specification!

Reply:
`Docker-Compose` (both capital) is definitely incorrect (should either be `docker-compose` (name of the binary), or `Docker Compose`). Perhaps avoid the name altogether and use something similar as the `docker` cli uses (`Print version information and quit`)

```suggestion
      version            Show version information and quit
```

(quit/exit, not sure what's clearer)? `curl` uses `quit`;

```
 -V, --version       Show version number and quit
```

Reply:
Sounds good! I could nitpick further but this is much better than what is already there. 😂

Reply:
Pushed your suggested changes in both places.

Reply:
Cool! LGTM